src/plugins/blockbook.ts

```diff
@@ -7,6 +8,12 @@ import { blockbookProtocol } from '../types/blockbookProtocol'
 
 const MAX_ADDRESS_COUNT_PER_CONNECTION = 100
 
+const pluginErrorCounter = new Counter({
+  name: 'websocket_error_event_count',
```

Comment:
The name should probably be `change_blockbook_error_count`. Then later we can use tags to identify specific types of errors, like websocket error events / blockbook protocol errors / etc.. We definitely want `change_blockbook_` at the start, to identify the scope (change server / blockbook plugin).

Reply:
Do you mean only for this counter name, or use the `change_blockbook_` prefix for all counter names for `blockbook.ts`?